third_party/move/move-compiler-v2/tests/reference-safety/v1-tests/borrow_field_full_invalid.exp

```diff
@@ -1,50 +1,14 @@
 
 Diagnostics:
-error: cannot copy mutable reference in local `c` which is still borrowed
-   ┌─ tests/reference-safety/v1-tests/borrow_field_full_invalid.move:15:9
-   │
-14 │         let f1 = &inner.f1;
-   │                  --------- previous field borrow
-15 │         *c;
-   │         ^^ copied here
-
-error: cannot copy mutable reference in local `inner` which is still borrowed
-   ┌─ tests/reference-safety/v1-tests/borrow_field_full_invalid.move:16:9
-   │
-14 │         let f1 = &inner.f1;
-   │                  --------- previous field borrow
-15 │         *c;
-16 │         *inner;
-   │         ^^^^^^ copied here
-
-error: cannot copy mutable reference in local `inner` which is still borrowed
-   ┌─ tests/reference-safety/v1-tests/borrow_field_full_invalid.move:23:18
-   │
-22 │         let c = id_mut(inner);
-   │                 ------------- previous mutable call result
-23 │         let f1 = &inner.f1;
-   │                  ^^^^^^^^^ copied here
-
-error: cannot immutable borrow local since other mutable references exist
+error: cannot immutable borrow local `inner` since other mutable references exist
```

Comment:
It would be nice to add comments to the test cases to explain why these should be ok.  Is it just that there are no writes through inner so we can consider it as not really mut?